mpeg-encoder.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function () {
```

Comment:
Do we need docs on how to use it?

Reply:
already documented https://github.com/ai/audio-recorder-polyfill#custom-encoder.
do you want to add mp3 example?

Reply:
Yeap, we need to add:

* That we have out-of-the-box MP3 support
* Guide with exact steps to enable MP3

Reply:
anyway, I've updated the readme with example how to use the new mpeg encoder.
https://github.com/ai/audio-recorder-polyfill/pull/43/commits/ad87bdb02489579a824717ce8ff6e9407cd489b2

Reply:
Great. I added a few notes to polish the docs

Reply:
I updated the readme https://github.com/ai/audio-recorder-polyfill/pull/43/files#diff-04c6e90faac2675aa89e2176d2eec7d8R144